python/lsst/ts/mtdomecom/mtdome_com.py

```diff
@@ -1092,29 +1098,19 @@ async def request_llc_status(self, llc_name: LlcName) -> None:
         ----------
         llc_name: `LlcName`
             The name of the lower level component.
-
-        Raises
-        ------
-        ValueError
-            In case the status dict received does not the `LlcName` key.
         """
         # Assume that the corresponding callback exists. The check for that is
         # in _start_periodic_tasks where the telemetry task is scheduled.
         cb = self.telemetry_callbacks[llc_name]
 
         command = CommandName(f"status{llc_name.value}")
-        try:
-            status: dict[str, typing.Any] = await self.write_then_read_reply(
-                command=command
-            )
-        except Exception as e:
-            await cb({"exception": e})  # type: ignore
-            return
-
-        if llc_name not in status:
-            raise ValueError(
-                f"No telemetry for subsystem {llc_name} Received {status=}."
-            )
+        status: dict[str, typing.Any] = {}
```

Comment:
```python
status: dict[str, typing.Any] = dict()
```

The `{}` can be set and dict in Python.

Reply:
The Python way of initializing dicts is to use `{}` so I leave it this way.